pkgs/tools/text/mdbook-plantuml/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, stdenv, rustPlatform, darwin, pkg-config, openssl
+, libiconv, CoreServices }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mdbook-plantuml";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "sytsereitsma";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1m53sp3k387injn6mwk2c6rkzw16b12m4j7q0p69fdb3fiqbkign";
+  };
+
+  cargoSha256 = "0xi14k86ym3rfz6901lmj444y814m7vp90bwsyjmcph3hdv6mjp0";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl.dev ]
```

```suggestion
  buildInputs = [ openssl ]
```
This should just work, or does it not?

---

it will, but openssl will output both `dev` and `bin`, this package only needs `dev`

---

That shouldn't make a big difference, I just want to avoid any compatibility problems and unexcepted things that may arise because we do that nowhere else really.